data/gegno/gegno missions.txt

```diff
@@ -0,0 +1,103 @@
+			label sacrifice
+			`	"The Gegno believe it to be necessary to sacrifice whatever must be, in order to imagine a future for themselves. Thus, they have separated into two different branches. One brandishes themselves as the honorable Vi- mightiest of warriors, toughest of kin. They are brutes that see to proof of strength being the one and only variable in determining their path. That of the Scin, the more technologically able, believe that only through progression of sciences and discovery, will the Gegno prosper."`
+
+			choice
+				`"The Vi are right, the strength to survive is more important than anything else." `
+					goto vi
+				`"The Scin are right, technological advancement opens up so many possibilities."`
+					goto scin
+				`"Why do they not combine their efforts to better their future together?"`
+					goto together
+			label vi
+			`	"While it is characteristic, such a tremendous proposition is rather dull for a growing species. Be that as it may, you humans follow to a much lesser degree than that of the Vi."`
+			label scin
+			`	The Quarg now shifts facial expressions in such a way to suggest a small hint of disappointment. "The pursuit of technology leaves behind many values important to a species' growth."`
+			label together
+			`	"Regrettably, the Gegno chiefly do not have a drive to pursue their future, diluting themeselves far into their present. Segregation of followings was one of few ways to seek motivation beyond the fact."`
+			choice
+				`	"It seems like you do not like the paths they've chosen."`
+			`	With its physique raising above you, the Quarg now stands up, darting its gaze off to the side. The verve it exhibited earlier is now long gone. "It is not in our grasp to persuade the Gegno, as once we did attempt to. They would ensure conflict with us should we interfere, and we have no desire to witness an uprisal of such a sort once more." It pauses, then gestures you to stand up as well. "Similarly, the Gegno do not wish to interact with audacious outsiders. Unfortunately, only a brief history of them we can tell. We cannot advocate pleasantries between you and them, though you are welcome at this ring should you find yourself here again."`
+			`	The Quarg motions a bow to you, and the first one from earlier escorts you out of the room. It seems associating with these so-called "Gegno" is very unlikely for you, and you will have to deal with their hostility for the foreseeable future."`
+				decline
+	on decline
+		log `Learned from the Quarg that the Gegno are not interested in interaction with outsider species.`
```

Comment:
Perhaps also add some faction log entries for the Gegno (perhaps as a whole, or both factions separately, or both factions and a more general entry) including some of what the Quarg tells us?

Reply:
This is done so on offer of the mission.